DEVDOC.md

```diff
@@ -1,5 +1,43 @@
 # GAPID Developer Documentation
 
+## Setup Golang development
+
+This project contains Golang code, but it does not have the file hierarchy of
+regular Golang projects (this is due to the use of Bazel as a build system).
+The `cmd/gofuse` utility enables to re-create the file hierarchy expected by Go
+tools:
+
+```sh
+# We assume GAPID has been checked out under ~/gapid
+cd ${HOME}/gapid
+# Make sure to build to have all compile-time generated files
+bazel build //pkg
```

Comment:
"bazel build pkg", again to be consistent with BUILDING.md. Users who are new to bazel get confused if "//" is required

Reply:
done